src/ray/raylet/worker_pool.h

```diff
@@ -156,6 +162,9 @@ class WorkerPool {
     /// A map from the pids of starting worker processes
     /// to the number of their unregistered workers.
     std::unordered_map<pid_t, int> starting_worker_processes;
+    /// A cache map for looking up the task id of actor creation task by the pid of
+    /// worker.
+    std::unordered_map<pid_t, TaskID> worker_to_task_id_cache;
```

Comment:
This only stores the mapping specifically for a specific usage, maybe consider renaming it?